sunpy/map/mapsequence.py

```diff
@@ -1,4 +1,4 @@
-"""A Python MapSequence Object"""
+"""A MapSequence (lmao) object"""
```

Comment:
?

Reply:
Its the new name, there was a meeting about it.